### PR TITLE
Fixing a bug where the context ends up in a bogus state when the iOS …

### PR DESCRIPTION
**Pull request by picHeidi**
…forces the queue onto a new thread.  Also removing unused and confusing blurFractionVaribales since they do nothing

### CODE REVIEW

**Comment by picHeidi** [comment]
@quasor This is a fix for swapping threads but keeping a gl context

File: framework/Source/GPUImageGaussianBlurFilter.h [hunk 1]
```diff
@@ -18,10 +18,6 @@
  */
 @property (readwrite, nonatomic) CGFloat blurRadiusInPixels;
 
-/** Setting these properties will allow the blur radius to scale with the size of the image
- */
-@property (readwrite, nonatomic) CGFloat blurRadiusAsFractionOfImageWidth;
-@property (readwrite, nonatomic) CGFloat blurRadiusAsFractionOfImageHeight;
```
Comment (quasor) [inline framework/Source/GPUImageGaussianBlurFilter.h]: Just making sure you want to include these blur changes.

**Comment by quasor** [comment]
otherwise look good to me.

**Comment by picHeidi** [comment]
Yeah, we were actually setting those variables - probably not realizing
they did .... nothing...

**Pull request merged**